Makefile

```diff
@@ -146,6 +148,37 @@ install_tfhe_lints:
 	(cd utils/cargo-tfhe-lints-inner && cargo install --path .) && \
 	cd utils/cargo-tfhe-lints && cargo install --path .
 
+.PHONY: setup_venv # Setup Python virtualenv for wasm tests
+setup_venv:
+	@python3 -m venv venv > /dev/null 2>&1 || echo "Virtualenv already exists"
+	@source venv/bin/activate && \
+	pip3 install -r ci/webdriver_requirements.txt
+
+# This is an internal target, not meant to be called on its own.
+install_web_resource:
+	wget -P $(dest) $(url)
+	@cd $(dest) && \
+	echo "$(checksum) $(filename)" > checksum && \
+	sha256sum -c checksum && \
```

Comment:
not needed here, but is there a way to load the checksumss and check a file from the terminal directly ?